.github/ISSUE_TEMPLATE/1_bug_report.yaml

```diff
@@ -0,0 +1,33 @@
+name: Bug report
+description: Report an error or unexpected behavior
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **Please review [our guide on interacting with the issue tracker](https://github.com/astral-sh/uv/issues/9452) before opening a new issue.**
+
+  - type: textarea
+    attributes:
+      label: Summary
+      description: |
+        A clear and concise description of the bug, including [a minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example).
+        If we cannot reproduce the bug, it is unlikely that we will be able to help you.
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Platform
+      description: What operating system and architecture are you using?
+      placeholder: e.g., macOS 14 arm64, Windows 11 x86_64, Ubuntu 20.04 amd64, ...
```

Comment:
Should we include Python version as a third field? Or otherwise incorporate that somehow?